### PR TITLE
Expand jobs aside

### PR DESCRIPTION
**Pull request by amyhughes**
This brings the jobs aside into view when the button is clicked:

### Before click
image

### After click
image

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -224,13 +222,17 @@ <h2>Colour system</h2>
...
       </p>
       <div class="jobs-footer">
-        The Guardian<br />
-        Design System
-        <button>Back</button>
+        <div>
+          The Guardian<br />
+          Design System
+        </div>
+        <button class="menu" href="#" id="jobs-aside-button">
```
Comment (SiAdcock) [inline index.html:229]: We can get rid of the `href`

File: scripts/index.js [hunk 2]
```diff
@@ -15,3 +13,22 @@ const updateHeaderHeight = () => {
 }
 
 window.addEventListener('scroll', updateHeaderHeight)
+
+const jobsAsideButton = document.getElementById("jobs-aside-button")
+const jobsAside = document.getElementById("jobs")
+
+// TODO: the button isn't clicking. Is the header hiding it?
```
Comment (SiAdcock) [inline scripts/index.js:20]: Oh yes it is! 😆

File: styles/jobs.css [hunk 3]
```diff
@@ -1,13 +1,49 @@
 .jobs {
   /* TODO: display this when the menu button is clicked */
```
Comment (SiAdcock) [inline styles/jobs.css:2]: Done!

**Review by SiAdcock — APPROVED** [review]
A couple of minor comments, but this is looking great! Thanks so much @amyhughes 💎

File: index.html (hunk 1 above)
Comment (amyhughes) [reply index.html]: Done, thanks!

**Pull request merged**